react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -37,12 +30,13 @@ export default class StopWatch extends React.Component {
   }
 
   render() {
+    this.state.switch === false ? this.image = '../images/play.png' : this.image = '../images/pause.png';
```

Comment:
Instead of tacking extra junk onto the `this` object, just make a variable

```js
const image = this.state.switch === false ? '../images/play.png' : '../images/pause.png';
```